template/src/ontology/Makefile.jinja2

```diff
@@ -1519,6 +1519,25 @@ validate-all-tsv: $(ALL_TSV_FILES)
 # Editors Utilities
 # ----------------------------------------
 
+# The merge template workflow is commonly used when ontology content is curated in a ROBOT template
+# with the intention to merge it directly into the editors file. The idea is to either create a template
+# manually or create a rule in {{ project.id }}.Makefile to download a template from, e.g., a Google Sheet, and then run
+# `sh run.sh make MERGE_TEMPLATE=tmp/merge_my_template.tsv merge_template` to merge the template into the editors file or
+# `sh run.sh make TEMPLATE_URL=http://example.org/merge_my_template.tsv merge_template` to download and then merge the template
+
+MERGE_TEMPLATE=tmp/merge_template.tsv
+MERGE_TEMPLATE_OPTIONS= --merge-before
+TEMPLATE_URL=
+
+# The tmp/merge_template.tsv is hardcoded as placeholder for downloading template files
+# from a remote location
+tmp/merge_template.tsv:
+	wget "$(TEMPLATE_URL)" -O $@
+
+merge_template: $(MERGE_TEMPLATE)
+	$(ROBOT) template $(MERGE_TEMPLATE_OPTIONS) --input $(SRC) \
+			--template $(MERGE_TEMPLATE) convert -f $(EDIT_FORMAT) -o $(SRC)
```

Comment:
Shouldn’t the `convert` part of this command includes a `--check false` option, in the case the edit format is OBO?

Reply:
I personally dont think so, no. It is horrible enough if obo release files are not really valid obo, but the editors file - no, I don't think there is an excuse. But I am willing to entertain counterarguments!